cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -887,7 +887,7 @@ def clean_conference_info_fields(v, required=True):
                     "value": clean_val("g", v, str),
                 }
             ],
-            "series": str(series_number),
+            "series": str(series_number).strip("[]"),
```

Comment:
why do you use the `.strip("[]")` ?

Reply:
because the `series_number` is an array e.g. `[2, 3]` and I want it to be a string like `"2, 3"` because in the schema of conference_info.series is a string